.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,12 @@
+blank_issues_enabled: true
```

Comment:
Blank issue are still supported. But we could also include an "Other" form for stuff that is not yet covered by Bugs and feature requests.